### PR TITLE
Create FilterTabs and FilterTab components 

### PR DESCRIPTION
**Pull request by kacperjanusz**

### CODE REVIEW

File: src/components/FilterTab/FilterTab.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Tab as MuiTab, TabProps as MuiTabProps, makeStyles } from '@material-ui/core';
+
+export type TabProps = MuiTabProps;
+
+const useStyles = makeStyles((theme) => ({
+  Tab: {
+    '&.MuiTab-textColorPrimary': {
+      color: theme.palette,
```
Comment (KlaraGajaszek) [inline src/components/FilterTab/FilterTab.tsx]: We should add sth more theme.palette...

File: src/components/Button/Button.stories.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
```
Comment (KlaraGajaszek) [inline src/components/Button/Button.stories.tsx]: Button component should be merged as a separate PR

File: src/components/FilterTab/FilterTab.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Tab as MuiTab, TabProps as MuiTabProps, makeStyles } from '@material-ui/core';
+
+export type TabProps = MuiTabProps;
+
+const useStyles = makeStyles((theme) => ({
+  Tab: {
```
Comment (KlaraGajaszek) [inline src/components/FilterTab/FilterTab.tsx]: Maybe lowercase => tab

File: src/components/FilterTabs/FilterTabs.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+import { Tabs as MuiTabs, TabsProps as MuiTabsProps, makeStyles } from '@material-ui/core';
+import { FilterTab } from '../FilterTab/FilterTab';
+export type TabsProps = MuiTabsProps;
+
+const useStyles = makeStyles((theme) => ({
```
Comment (KlaraGajaszek) [inline src/components/FilterTabs/FilterTabs.tsx]: I think you don't need it

File: src/components/FilterTabs/FilterTabs.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { FilterTabs, TabsProps } from '.';
+
+export default {
```
Comment (KlaraGajaszek) [inline src/components/FilterTabs/FilterTabs.stories.tsx:5]: I think, you should add component: FilterTabs

**Review by KlaraGajaszek — CHANGES_REQUESTED** [review]

File: src/components/FilterTabs/FilterTabs.tsx [hunk 6]
```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+import { Tabs as MuiTabs, TabsProps as MuiTabsProps, makeStyles } from '@material-ui/core';
+import { FilterTab } from '../FilterTab/FilterTab';
+export type TabsProps = MuiTabsProps;
+
+export const FilterTabs = (props: TabsProps): JSX.Element => {
+  const [value, setValue] = useState(1);
+
+  const handleChange = (event: React.ChangeEvent<{}>, newValue: number) => {
```
Comment (KonradSzwarc) [inline src/components/FilterTabs/FilterTabs.tsx]: Use type from the library.
`const handleChange: MuiTabsProps ['onChange']` = (event, newValue) => {

File: src/components/FilterTabs/FilterTabs.tsx [hunk 7]
```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+import { Tabs as MuiTabs, TabsProps as MuiTabsProps, makeStyles } from '@material-ui/core';
+import { FilterTab } from '../FilterTab/FilterTab';
+export type TabsProps = MuiTabsProps;
```
Comment (KonradSzwarc) [inline src/components/FilterTabs/FilterTabs.tsx]: Add a new line between imports and types

File: src/components/FilterTab/FilterTab.tsx [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+      color: theme.palette.secondary,
+    },
+
+    '&.MuiTab-textColorPrimary.Mui-selected': {
+      color: 'theme.palette.primary.main',
+    },
+  },
+}));
+
+export const FilterTab = (props: TabProps): JSX.Element => {
```
Comment (KonradSzwarc) [inline src/components/FilterTab/FilterTab.tsx]: You don't need to specify return type explicitly

**Review by KonradSzwarc — CHANGES_REQUESTED** [review]

**Review by KlaraGajaszek — APPROVED** [review]

**Review by KonradSzwarc — APPROVED** [review]

**Pull request merged**